indexing-service/src/test/java/org/apache/druid/indexing/compact/OverlordCompactionSchedulerTest.java

```diff
@@ -158,7 +158,10 @@ public void testCompactionSupervisorConfigSerde() throws JsonProcessingException
             OBJECT_MAPPER.writeValueAsString(ImmutableMap.of("enabled", enabled, "engine", defaultEngine)),
             CompactionSupervisorConfig.class
         );
-    Assert.assertEquals(new CompactionSupervisorConfig(enabled, defaultEngine), compactionSupervisorConfig);
+    CompactionSupervisorConfig expectedConfig = new CompactionSupervisorConfig(enabled, defaultEngine);
+    Assert.assertEquals(expectedConfig, compactionSupervisorConfig);
+    // Superfluous test to meet min branch coverage for CompactionSupervisorSpec.
+    Assert.assertNotEquals(expectedConfig, "enabled");
```

Comment:
Nit: Best to have this test in a separate method.
Let's not call it superfluous as it does verify some code after all 😛 

```suggestion
    Assert.assertNotEquals(expectedConfig, "enabled");
```

Reply:
Fixed!